pkg/provider/loadbalancer/securitygroup/securitygroup.go

```diff
@@ -379,41 +379,6 @@ func (helper *RuleHelper) removeDestinationFromRule(rule *network.SecurityRule,
 	return helper.addAllowRule(rule.Protocol, ipFamily, ListSourcePrefixes(rule), prefixes, expectedPorts)
 }
 
-// RemoveDestinationPrefixesFromRules removes the given destination addresses from all rules.
-func (helper *RuleHelper) RemoveDestinationPrefixesFromRules(prefixes []string) {
```

Comment:
Is this function only used in tests?

Reply:
Yes, it has been replaced with `RemoveDestinationFromRules` in LC306